src/Sulu/Bundle/AdminBundle/Resources/js/views/Datagrid/toolbarActions/AddToolbarAction.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import {translate} from '../../../utils/Translator';
+import AbstractToolbarAction from './AbstractToolbarAction';
+
+export default class AddToolbarAction extends AbstractToolbarAction {
+    getToolbarItemConfig() {
+        return {
+            icon: 'su-plus-circle',
+            label: translate('sulu_admin.add'),
+            onClick: this.datagrid.handleItemAdd,
```

Comment:
I also think that this is a little bit strange... For me also `this.datagrid.addItem` or something similar would make more sense here. If there is a reference to a `handle` method I always assume that it is in the same class...